.github/workflows/downstream.yml

```diff
@@ -0,0 +1,78 @@
+name: Downstream checks
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, 'please test downstream') }}
+
+jobs:
+  dqlite:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install apt deps
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -qq automake libtool gcc make liblz4-dev libuv1-dev libsqlite3-dev
+
+      - name: Check out raft
+        uses: actions/checkout@v3
+        with:
+          ref: refs/pull/${{ github.event.issue.number }}/head
+          path: raft
+
+      - name: Install raft
+        run: |
+          cd raft
+          autoreconf -i
+          ./configure --enable-debug --enable-uv --enable-sanitize
+          sudo make -j$(nproc) install
+          sudo ldconfig
+
+      - name: Check out dqlite
+        uses: actions/checkout@v3
+        with:
+          repository: canonical/dqlite 
+          path: dqlite
+
+      - name: Test and install dqlite
+        run: |
+          cd dqlite
+          autoreconf -i
+          ./configure --enable-debug --enable-sanitize
+          sudo make -j$(nproc) check || (cat ./test-suite.log && false)
+          sudo make install
+          sudo ldconfig
+
+      - name: Install Go
+        uses: actions/setup-go@v4
+
+      - name: Check out go-dqlite
+        uses: actions/checkout@v3
+        with:
+          repository: canonical/go-dqlite
+          path: go-dqlite
+
+      - name: Test go-dqlite
+        env:
+          GO_DQLITE_MULTITHREAD: '1'
+        run: |
+          cd go-dqlite
+          go get -tags libsqlite3 -t ./...
+          go test -asan -v -race ./...
+          VERBOSE=1 DISK=${{ matrix.disk }} ./test/dqlite-demo.sh
```

Comment:
I presume the `{{ matrix.disk }}` token expands to empty string and can be dropped?